code/datums/sound_player.dm

```diff
@@ -215,7 +215,7 @@ datum/sound_token/proc/Mute()
 	status = new_status
 	PrivUpdateListeners()
 
-/datum/sound_token/proc/PrivAddListener(var/atom/listener)
+/datum/sound_token/proc/PrivAddListener(var/mob/listener)
```

Comment:
I don't think this change makes sense
isn't this system designed to accomodate simulating objects recieving sounds? 
clients could also be passed in